cmd/ceremony/README.md

```diff
@@ -266,6 +267,57 @@ ocsp-profile:
 
 This config generates a OCSP response signed by a key in the HSM, identified by the object label `root signing key` and object ID `ffff`. The response will be for the certificate in `/home/user/certificate.pem`, and will be written to `/home/user/ocsp-resp.b64`.
 
+### CRL ceremony
+
+- `ceremony-type`: string describing the ceremony type, `crl`.
+- `pkcs11`: object containing PKCS#11 related fields.
+    | Field | Description |
+    | --- | --- |
+    | `module` | Path to the PKCS#11 module to use to communicate with a HSM. |
+    | `pin` | Specifies the login PIN, should only be provided if the HSM device requires one to interact with the slot. |
+    | `signing-key-slot` | Specifies which HSM object slot the signing key is in. |
+    | `signing-key-label` | Specifies the HSM object label for the signing key. |
+    | `signing-key-id` | Specifies the HSM object ID for the signing key. |
+- `inputs`: object containing paths for inputs
+    | Field | Description |
+    | --- | --- |
+    | `issuer-certificate-path` | Path to PEM issuer certificate. |
+- `outputs`: object containing paths to write outputs.
+    | Field | Description |
+    | --- | --- |
+    | `crl-path` | Path to store signed PEM CRL. |
+- `crl-profile`: object containing profile for the CRL.
+    | Field | Description |
+    | --- | --- |
+    | `this-update` | Specifies the CRL thisUpdate date, in the format `2006-01-02 15:04:05`. The time will be interpreted as UTC. |
+    | `next-update` | Specifies the CRL nextUpdate date, in the format `2006-01-02 15:04:05`. The time will be interpreted as UTC. |
+    | `number` | Specifies the CRL number. Each CRL should have a unique monotonically increasing number. |
+    | `revoked-certificates` | Specifies any revoked certificates that should be included in the CRL. May be empty. If present it should be a list of objects with the fields `certificate-path`, containing the path to the revoked certificate, and `revocation-date`, containing the date the certificate was revoked, in the format `2006-01-02 15:04:05`. |
```

Comment:
This should include a way to specify reason for revocation. Per the Microsoft Root Program Requirements (https://docs.microsoft.com/en-us/security/trusted-root/program-requirements#communication-and-disclosure-requirements):

> Reason Code must be included in revocations for intermediate certificates. CAs must update the CCADB when revoking any intermediate certificates within 30 days.

This is also incorporated into the CA/Browser Forum ballot SC31 - Browser Alignment.